ls1intum/Artemis
# [Bugfix] Fix position of alert on lecture pages

### Checklist
#### General



### Motivation and Context


The Alerts, e.g. die outdated alert, were not placed correctly on lecture pages. In particular, it was different to the exercise pages, where it looked fine. I re-placed the alert text in `course-lecture-details.component.hmtl` to be consistent with `course-exercise-details.component.html`.

### Steps for Testing


We use Metis features to create alerts, but this applies to every other alert as well. Feel free to trigger other alerts instead.

1. Log in to Artemis
2. Navigate to the lecture page
3. Delete or create a post -> see where the alert pops up, should be below the lecture information header
4. Navigate to the exercise page
5. Delete or create a post -> see where the alert pops up, should be below the exercise information header (nothing changed here)

### Review Progress





#### Code Review
#### Manual Tests

### Test Coverage






### Screenshots


How it **was** on lecture page:
<img width="1279" alt=" " src="https://user-images.githubusercontent.com/41366522/135413176-4042f44f-d174-4fe4-8aa6-c0c584f75283.png">


How it was and is on exercise pages: -> should now be the same fore lecture page
<img width="1257" alt="Bildschirmfoto 2021-09-30 um 09 02 11" src="https://user-images.githubusercontent.com/41366522/135413131-6caf1bbf-51d0-4ba4-82ef-52ac88439345.png">
